packages/devtools_app/lib/src/screens/debugger/codeview.dart

```diff
@@ -376,7 +372,7 @@ class _CodeViewState extends State<CodeView>
                                   controller: horizontalController,
                                   child: SizedBox(
                                     height: constraints.maxHeight,
-                                    width: fileWidth,
+                                    width: constraints.maxWidth,
```

Comment:
I think this should be the max of fileWidth and constraints.maxWidth so that we cover both the case where the file is wider than the view and more narrow than the view

Reply:
Ah yes, that makes more sense.